backend/DatabaseScripts/SqlScripts/0038_CreateView_graphql_contracts_view.sql

```diff
@@ -0,0 +1,108 @@
+﻿ALTER TABLE graphql_transaction_events
+	ALTER COLUMN event SET DATA type jsonb;
+CREATE INDEX event_tag on graphql_transaction_events (((event->>'tag')::INTEGER));
+CREATE INDEX event_contract_init_contract_address ON graphql_transaction_events (((event->'data'->>'ContractAddress'))) WHERE (event->>'tag')::INTEGER = 16;
+CREATE INDEX event_contract_upgrade_contract_address ON graphql_transaction_events (((event->'data'->>'ContractAddress'))) WHERE (event->>'tag')::INTEGER = 36;
+CREATE INDEX event_contract_update_contract_address ON graphql_transaction_events (((event->'data'->>'ContractAddress'))) WHERE (event->>'tag')::INTEGER = 18;
```

Comment:
What are these tags? Are they event types? Can we document here. this is pretty hard to understand/maintain.

Reply:
these are the tags the event is serialized with
https://github.com/Concordium/concordium-scan/blob/71cde2ee2f8635a008c170fadbf89543ef19b2a4/backend/Application/Api/GraphQL/EfCore/Converters/Json/TransactionResultEventConverter.cs#L9-L44